libs/forklift_inventory.py

```
@@ -392,15 +405,15 @@ def vms_storages_mappings(self, vms: list[str]) -> list[dict[str, str]]:
 
         return _mappings
 
-    def vms_networks_mappings(self, vms: list[str]) -> list[dict[str, str]]:
+    def vms_networks_mappings(self, vms: list[str], deduplicate: bool = True) -> list[dict[str, str]]:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)